### ISSUE TITLE
Comments are left out from screenshot regardless of settings

### ISSUE THREAD

**Issue by hckr** [hckr/fb-post-screenshot#28]
It seems solution for #22 now causes problems for all posts (maybe only on some machines / facebook profiles), so it needs to be reverted. It no longer works for video either, but the revert will break this use case further (no screeshot will be created). This will have to be handled later.

**Comment by Welkin333** [comment]
Hi hckr. Thanks for the great app. It has been working wonderful for the last couple of months. We are unfortunately experiencing the same issue now. We have been using it to capture Facebook posts related to COVID-19. Looking forward to the fix so we can continue capturing these posts.

**Comment by hckr** [comment]
I'll keep that in mind. Could you describe those issues you're facing? Is it connected to this issue in particular or is it something different?

**Comment by Welkin333** [comment]
Hi hckr. It was related to this issue. I have been using it steadily. This issue only came up a couple of weeks ago. When I click the save screenshot, the comments are not saved. The image is still saved, but with no comments. I tried it again today, and it does not appear to be even saving the post anymore.

**Comment by hckr** [comment]
Sorry, I'm quite busy recently. I'll look into this ASAP.

**Comment by Welkin333** [comment]
Hi Jakub,

Thanks for looking into this issue. I did see that someone posted that they installed an older version of the app and it helped fix a similar issue. I did the same, and it seems to have worked for me. I am using version 1.10.0. I will stick with this version for now.

Hopefully this helps determine what is wrong with the newer version.

All the best,

William Vinh-Doyle
Digital Archivist / Archiviste des documents numériques
Provincial Archives of New Brunswick / Archives provinciales du Nouveau-Brunswick
Office of the Chief Information Officer / Bureau du chef de l’information
Finance and Treasury Board / Finances et Conseil du Trésor
Government of New Brunswick / Gouvernement du Nouveau-Brunswick

Phone / Téléphone : (506) 476-8904
E-mail / Courriel : William.Vinh-Doyle@gnb.ca<mailto:William.Vinh-Doyle@gnb.ca>
www.gnb.ca<http://www.gnb.ca/>
